packages/dart_frog_cli/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.3.8
+
+- feat: update brick bundles to dart 3.0.0
+- feat: Add dart-vm-port option to dev command
+- feat: added default port number for --dart-vm-service-port
```

Comment:
```suggestion
- feat: added default port number for --dart-vm-service-port
- feat: recommend using --dart-vm-service-port on VM port failure
```

https://github.com/VeryGoodOpenSource/dart_frog/pull/698 accidentally got merged before adding https://github.com/VeryGoodOpenSource/dart_frog/pull/697, can we add this to the changelog?